pkg/api/server/v1alpha2/auth/rbac_test.go

```diff
@@ -39,8 +39,15 @@ import (
 func TestRBAC(t *testing.T) {
 	// Map of users -> tokens. The 'authorized' user has full permissions.
 	users := map[string]string{
-		"authorized":   "a",
-		"unauthorized": "b",
+		"authorized":         "a",
+		"unauthorized":       "b",
+		"authorized-group":   "c",
+		"unauthorized-group": "d",
+	}
+	// Map of users -> groups. The 'authorized' user has full permissions.
+	groups := map[string][]string{
+		"authorized-group":   {"authorized"},
+		"unauthorized-group": {"unauthorized"},
 	}
```

Comment:
Are you able to add a test for the extras, too? I don't think we have coverage for this yet.

Reply:
Yes, I have added tests for extra now.